From 448547f8cb7faa7a72748c64f298f59392e1a188 Mon Sep 17 00:00:00 2001
From: Duncan Macleod <macleoddm@cardiff.ac.uk>
Date: Wed, 20 Aug 2025 09:22:39 +0100
Subject: [PATCH] Remove unneeded requirements

In conda-forge pesummary can depend on the individual LALSuite subpackages

Deepdish support is being dropped, and pytables is unused
---
 pyproject.toml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 0b388ce6..ece08a24 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -26,10 +26,8 @@ requires-python = ">=3.10"
 dependencies = [
 	"astropy>=5.0",
 	"corner",
-	"deepdish",
 	"gwpy >=3.0.11",
 	"h5py",
-	"lalsuite >7.11",
         "python-ligo-lw <2.0.0",
 	"ligo-gracedb",
 	"matplotlib >= 3.6.0,<3.10.0",
@@ -41,7 +39,6 @@ dependencies = [
 	"seaborn>=0.12.2,<0.13.0",
 	"statsmodels",
 	"scipy >=1.8.0",
-	"tables",
 	"tqdm >=4.44.0",
 ]
 dynamic = ['version']
-- 
2.39.5

